extensions/amp-addthis/validator-amp-addthis.protoascii

```diff
@@ -0,0 +1,73 @@
+#
+# Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-addthis
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-addthis"
+    allowed_versions: "0.1"
+    allowed_versions: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+
+tags: {  # <amp-addthis>
+  html_format: AMP
+  tag_name: "AMP-ADDTHIS"
+  requires_extension: "amp-addthis"
+  attrs: {
+    name: "data-pub-id"
```

Comment:
Up to you, not required:

If the pub-id follows a certain format then it's possible to restrict the set of values that can be here to follow that format via `value_regex_casei`.

Same with `data-widget-id` below.